### PR TITLE
Bump chunk-manifest-webpack-plugin version

### PR DESCRIPTION
**Pull request by chrisui**
To support webpack 3 in peer dependencies. Would be great to get this merged and published asap as it is a blocker for upgrading to webpack 3.

Resolves https://github.com/jouni-kantola/inline-chunk-manifest-html-webpack-plugin/issues/16

*Kept the tilde range but a more typical semver caret range might be more appropriate in future.*

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "prettier": "^1.1.0"
   },
   "dependencies": {
-    "chunk-manifest-webpack-plugin": "~1.0.0"
+    "chunk-manifest-webpack-plugin": "~1.1.1"
```
Comment (jouni-kantola) [inline package.json]: Might as well patch to 1.1.2 as there apparently was a regression bug in 1.1.1. See below:
https://github.com/soundcloud/chunk-manifest-webpack-plugin/commit/1321345aa97c8f944f6f8f7cf3e3dbfff5eea31b

So I'd suggest `~1.1.2`.

**Review by jouni-kantola — CHANGES_REQUESTED** [review]
I've checked the commits in `chunk-manifest-webpack-plugin` and I guess this looks OK if `~1.1.2` is used.

Before publishing a new version of `inline-chunk-manifest-html-webpack-plugin` I think I'd want to pass in explicitly to not inline with chunk-manifest-webpack-plugin, but that can be done in a separate PR.

**Comment by jouni-kantola** [comment]
@chrisui, please have a look at this comment:
https://github.com/jouni-kantola/inline-chunk-manifest-html-webpack-plugin/issues/16#issuecomment-318186916

**Comment by chrisui** [comment]
@jouni-kantola Thanks! Just updated the version number.

**Comment by chrisui** [comment]
@jouni-kantola any chance this could be merged and published soon? Sorry to pester, just really excited to get webpack 3 setup at work!

**Comment by jouni-kantola** [comment]
I don't want to updare until this PR https://github.com/soundcloud/chunk-manifest-webpack-plugin/pull/51 is merged and a new version of `chunk-manifest-webpack-plugin` is published.

Sorry for the delayed answer. I'm occupied with newborn twins 🍼

**Comment by chaffeqa** [comment]
You guys could use `@codemotion/chunk-manifest-webpack-plugin` in the interim

See https://github.com/soundcloud/chunk-manifest-webpack-plugin/issues/47#issuecomment-331461628

**Comment by jouni-kantola** [comment]
Thank you for the help, @chrisui. I now took the time to fork [chunk-manifest-webpack-plugin](https://github.com/soundcloud/chunk-manifest-webpack-plugin/). I'm bumping the version number to 2.0.0, as I'm dropping webpack 1 support (I don't even know if I've ever tested in webpack 1).

In favour of merging PR https://github.com/jouni-kantola/inline-chunk-manifest-html-webpack-plugin/pull/18, I'm closing this one. I've now set webpack 2/3 as peer dependency.

I just published [2.0.0 to npm](https://www.npmjs.com/package/inline-chunk-manifest-html-webpack-plugin).

**Pull request closed without merging**